snappy-core/src/main/scala/org/apache/spark/sql/SnappyContext.scala

```diff
@@ -796,17 +797,21 @@ object GlobalSnappyInit {
         // prior to `new SnappyContext(sc)` after this
         // method ends.
         ToolsCallbackInit.toolsCallback.invokeLeadStartAddonService(sc)
+        SnappyAnalyticsService.start(sc)
```

Comment:
We can put another class like SnappyDaemons.start() . Inside it we can start SnappyAnalyticService. As tomorrow this list might grow :)